docs/source/pickling_cuml_models.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>You might want to give a bit more context for why we're interested in this at this point. As is, it reads as something of a non-sequitur.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/3154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Yeah this particular cell can be removed. I had it in the notebook originally for demonstrating distributed ML capabilities for folks. </p>

Reply:
@wphicks, I've cleaned it up quite a bit so that now the single- and multi-GPU examples are much easier to compare side by side. Mind giving it another look when you have a moment?